h2o-py/h2o/automl/_estimator.py

```diff
@@ -73,19 +73,29 @@ class H2OAutoML(H2OAutoMLBaseMixin, Keyed):
     Automatic Machine Learning
 
     The Automatic Machine Learning (AutoML) function automates the supervised machine learning model training process.
-    The current version of AutoML trains and cross-validates the following algorithms (in the following order):
-    three pre-specified XGBoost GBM (Gradient Boosting Machine) models,
-    a fixed grid of GLMs,
-    a default Random Forest (DRF),
-    five pre-specified H2O GBMs,
-    a near-default Deep Neural Net,
-    an Extremely Randomized Forest (XRT),
-    a random grid of XGBoost GBMs,
-    a random grid of H2O GBMs,
-    and a random grid of Deep Neural Nets.
```

Comment:
@ledell I updated this description as it is outdated now. Feel free to rephrase.